Makefile

```diff
@@ -209,7 +211,11 @@ endif
 # UTILITIES
 # =========
 
-fmt:
+imports: 
+	@$(call print, "Fixing imports.")
+	goimports -w $(GOFILES_NOVENDOR)
+
+fmt: imports
```

Comment:
`goimports` will also run `gofmt`, so should we just replace it?

Reply:
Good observation, will look into it.

Reply:
Looks like `goimports` doesn't support code simplification, like we add here: https://github.com/lightningnetwork/lnd/pull/2118

Reply:
Should we just run both in `make fmt`?

Reply:
As in don't define `make imports`?

Reply:
Yep, agree it's better to keep all formatting tools under one target (`make fmt`).

Reply:
Makes sense, done!